### PR TITLE
Ltd 4209 update python to 3.8

### PR DESCRIPTION
**Pull request by Tllew**
### Aim
Upgrade python to 3.8.X

[LTD-4209](https://uktrade.atlassian.net/browse/LTD-4209)

[LTD-4209]: https://uktrade.atlassian.net/browse/LTD-4209?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -56,23 +55,23 @@ pypdf2 = "~=1.27.5"
 cryptography = "~=41.0.3"
 sentry-sdk = "~=1.17.0"
 elastic-apm = "~=6.7.2"
-gunicorn = "~=20.1.0"
-gevent = "~=20.12.1"
+gunicorn = "*"
```
Comment (kevincarrogan) [inline Pipfile:58]: How comes we're settings a wildcard version here?

**Review by depsiatwal — COMMENTED** [review]
any reason why we got 
code changes along with python upgrade 
i.e 
api/applications/views/applications.py

I would separate out the two

**Comment by kevincarrogan** [comment]
> any reason why we got code changes along with python upgrade i.e api/applications/views/applications.py
> 
> I would separate out the two

These are for linting errors. Without these changes this can't be merged as the linter complains.

**Review by depsiatwal — APPROVED** [review]
LGTM please test on env first 
which I see is in flight based on chats

**Review by hnryjmes — APPROVED** [review]

**Pull request merged**